types/mongodb/index.d.ts

```diff
@@ -962,8 +963,10 @@ export interface WriteOpResult {
 //http://mongodb.github.io/node-mongodb-native/2.1/api/Cursor.html#~resultCallback
 export type CursorResult = any | void | boolean;
 
+type Default = { [key: string]: any };
+
 //http://mongodb.github.io/node-mongodb-native/2.1/api/Cursor.html
-export class Cursor<T> extends Readable {
+export class Cursor<T = Default> extends Readable {
```

Comment:
Don't forget to do the same for ``AggregationCursor<T>``.
And if so we have the default generic types now the overriding methods of ``Collection`` like ``find()`` that return ``Cursor<any>`` and ``AggregationCursor<any>`` now left unnecessary so we have to remove them to avoid scope bloating.